org.eclipse.winery.topologymodeler.ui/package.json

```diff
@@ -1,6 +1,6 @@
 {
-    "name": "org.eclipse.winery.topologymodeler.ui",
-    "version": "2.0.0-SNAPSHOT",
+    "name": "topologymodeler",
+    "version": "2.1.0",
```

Comment:
MUST be the same as the root project - please revert the changes.

Reply:
done